src/orion/core/cli/frontend.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# pylint: disable=too-few-public-methods
+"""
+Web application endpoint
+========================
+
+Starts an http endpoint to serve requests
+
+"""
+import logging
+import mimetypes
+import os
+import site
+import sys
+
+import falcon
+from gunicorn.app.base import BaseApplication
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+DESCRIPTION = "Starts Oríon Dashboard"
+
+
+def get_dashboard_build_path():
+    """Find dashboard build folder.
+
+    If package is installed, dashboard build should be in installation prefix
+    https://docs.python.org/3/distutils/setupscript.html#installing-additional-files
+    Otherwise, dashboard build should be in dashboard folder near src
+    in orion repository.
+    """
+    current_file_path = __file__
+    if current_file_path.startswith(sys.prefix):
+        dashboard_build_path = os.path.join(sys.prefix, "orion-dashboard", "build")
+    elif current_file_path.startswith(site.USER_BASE):
+        dashboard_build_path = os.path.join(site.USER_BASE, "orion-dashboard", "build")
+    else:
+        dashboard_build_path = os.path.abspath(
+            os.path.join(
+                os.path.dirname(current_file_path),
+                "..",
+                "..",
+                "..",
+                "..",
+                "dashboard",
+                "build",
+            )
+        )
+    assert os.path.isdir(dashboard_build_path)
+    assert os.path.isfile(os.path.join(dashboard_build_path, "index.html"))
```

Comment:
Why should we verify this? Is it likely to occur? Do you think this would be best as a part of the tests instead?